client/components/theme-type-badge/test/index.jsx

```diff
@@ -43,8 +44,12 @@ describe( 'ThemeTypeBadge', () => {
 
 	describe( 'Premium theme popover', () => {
 		test( 'Free site', async () => {
+			const queryClient = new QueryClient();
+
 			const { container } = renderWithState(
-				<ThemeTypeBadge siteId={ 123 } siteSlug="example.com" themeId="premium/test" />
+				<QueryClientProvider client={ queryClient }>
```

Comment:
I think it'd be easier if you just mock `useSiteGlobalStylesStatus`

Reply:
Changed